lib/list.js

```diff
@@ -74,5 +74,27 @@
 		}
 	});
 
+  //search function
+  document.getElementById("search").addEventListener('keyup',function() {
+
```

Comment:
remove blank line, consider use var in place of let for consistency.  If we want to upgrade to use ES6+ syntax let's do that everywhere as separate issue.